scripts/local-dev.sh

```diff
@@ -50,16 +52,19 @@ parse_args() {
     DEFAULT_NAMESPACE=""
     LLAMA_MODEL_NAMESPACE=""
     MODEL_CONFIG_SOURCE="local"  # Default to local
+    LLM_MODEL=""  # Optional LLM model for config generation
 
     # Parse standard arguments using getopts
-    while getopts "n:N:m:M:c:C:" opt; do
+    while getopts "n:N:m:M:c:C:l:L:" opt; do
         case $opt in
             n|N) DEFAULT_NAMESPACE="$OPTARG"
                  ;;
             m|M) LLAMA_MODEL_NAMESPACE="$OPTARG"
                  ;;
             c|C) MODEL_CONFIG_SOURCE="$OPTARG"
                  ;;
+            l|L) LLM_MODEL="$OPTARG"
```

Comment:
Allows passing a new param (`-l / -L`) to specify a different LLM_MODEL in case the default model is not used for deployment.